react/components/molecules/keyboard/keyboard.js

```diff
@@ -39,13 +41,51 @@ export class Keyboard extends Component {
         super(props);
 
         this.state = {
-            symbolicKeyboard: this.props.symbolicKeyboard
+            symbolicKeyboard: this.props.symbolicKeyboard,
+            animationPositionY: new Animated.Value(212),
+            animationOpacity: new Animated.Value(0)
         };
 
         this.keys = "1234567890qwertyuiopasdfghjklzxcvbnm";
         this.keyColumns = [10, 10, 9, 9];
+        this.keyboardHiddenPositionY = 212;

```

Comment:
just a small thing, but perhaps use this variable in the  initial value of the `animationPositionY` above